go.mod

```diff
@@ -51,11 +51,10 @@ require (
 	github.com/mattn/go-isatty v0.0.20 // indirect
 	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
-	github.com/oapi-codegen/oapi-codegen/v2 v2.4.1 // indirect
```

Comment:
Oh now I understand the problem, Konflux does not run `prepare-sources.sh`! OMG why did I spent time investigating this. :-)